src/audio/session_config.c

```diff
@@ -19,11 +19,16 @@ const TempoData gTempoData = {
 #define AI_BUFFERS_SIZE (AIBUF_SIZE * ARRAY_COUNT(gAudioCtx.aiBuffers))
 #define SOUNDFONT_LIST_SIZE (NUM_SOUNDFONTS * sizeof(SoundFont))
 #define PERMANENT_POOL_SIZE (SFX_SEQ_SIZE + SFX_SOUNDFONT_1_SIZE + SFX_SOUNDFONT_2_SIZE)
+#if PLATFORM_N64
+#define EXTRA_SIZE 0x10
+#else
+#define EXTRA_SIZE 0
+#endif
```

Comment:
wondering if it makes sense to say in the name of the thing that it is specifically for n64 `N64_EXTRA_SIZE` or `N64_PADDING`. just so you dont have to click on the define to see the platform ifdef. but dont care too much

Reply:
and either way it may be good to make it more specific, by saying `N64_AUDIO_BUFFER_PADDING` or something along those lines

Reply:
hmm not sure that would save me a click personally. I really don't know what this is so it's hard to be more specific.

We could also do away with EXTRA_SIZE entirely and instead do something like
```
#if PLATFORM_N64
#define PERMANENT_POOL_SIZE (SFX_SEQ_SIZE + SFX_SOUNDFONT_1_SIZE + SFX_SOUNDFONT_2_SIZE + 0x10)
#else
#define PERMANENT_POOL_SIZE (SFX_SEQ_SIZE + SFX_SOUNDFONT_1_SIZE + SFX_SOUNDFONT_2_SIZE)
#endif
```

Reply:
im not sure if this is better either. But I do think the define name needs to be more specific than just EXTRA_SIZE if it stays

Reply:
I got rid of `EXTRA_SIZE`, because otherwise the `// permanent pool size` did not seem accurate, and now we don't have to give it a name